Carthage support via cocoapods

dostrander opened the pull request:
Used a suggestion by @ZevEisenberg to use cocoapods to set up the project. Made it much nicer...

dostrander commented:
@arrouse @nbonatsakis @mattThousand

ZevEisenberg commented on RZImport/.gitignore:
```diff
@@ -0,0 +1,33 @@
+# http://guides.cocoapods.org/using/using-cocoapods.html#should-i-ignore-the-pods-directory-in-source-control
+# 
+# Note: if you ignore the Pods directory, make sure to uncomment
+# `pod install` in .travis.yml
```
^ this (although I would check in pods, personally)

dostrander committed:
Carthage Support
- Minimal carthage support with just seperate symlinked `.xcodeproj`
- Changed podspec to get rid of Nick D contact and changed to @arrouse
- Added umbrella header that doesn't do anything, maybe want to structure the project to use it one day

dostrander committed:
Carthage Support
- Add `_private.h` header to public headers for `RZVinyl` support. This should be changed …
